### PR TITLE
Added confirmation before binding wallet-rpc to external IP

### PR DESCRIPTION
**Pull request by vtnerd**

### CODE REVIEW

**Comment by fluffypony** [comment]
Please rebase

File: src/wallet/wallet_rpc_server.cpp [hunk 1]
```diff
@@ -84,20 +87,35 @@ namespace tools
     return epee::http_server_impl_base<wallet_rpc_server, connection_context>::run(1, true);
   }
   //------------------------------------------------------------------------------------------------------------------------------
-  bool wallet_rpc_server::handle_command_line(const boost::program_options::variables_map& vm)
```
Comment (moneromooo-monero) [inline src/wallet/wallet_rpc_server.cpp]: Could also be removed from the class definition.

**Comment by moneromooo-monero** [comment]
ok, minor but trivial comment in the source

**Comment by fluffypony** [comment]
Merging; feel free to PR separately for @moneromooo-monero's small note

**Pull request merged**